app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,162 @@
+
+To customize your build pipeline, you can use `polymer-build` itself in conjunction with other build tools.
+
+## polymer init
+Initializes a Polymer project. Contains a set of starter templates for elements and apps. 
+
+## polymer install
+Install tool for Polymer apps. Installs [bower](#bower) dependencies, equivalent to running `bower install`. With the option `--variants`, also installs any [dependency variants](#dependency-variants) specified in the `variants` property of [`bower.json`](#bower-json).
+
+## polymer lint
+Invokes [`polylint`](#polylint) to [lint](#linter) a polymer project.
+
+## polymer serve
+This command starts [`polyserve`](#polyserve), a simple development server for Web Components.
+
+## polymer test
+Runs tests for your Polymer app with [`web-component-tester`](#web-component-tester).
+
+## polymer-analyzer
+A static analysis framework for web applications. 
+
+## polymer-cli
+A command line tool for Polymer projects. `polymer-cli` has a number of tools for working with Polymer and Web Components.
+
+## polymer-linter
+`polymer-linter` is in development. It will be a tool for [linting](#linter) Polymer applications, and will replace [`polylint`](#polylint). For a working linter, use `polylint`.
+
+## `polymer.json`
+`polymer.json` is the Polymer build configuration file. It lets you configure your app’s entrypoint and app shell files, instead of passing these as parameters to polymer build.
+
+Polymer.json also lets you specify the locations of source files, HTML fragments and additional dependencies.
+
+## `polyserve`
+`polyserve` is a locally-installable web server for developing [bower](#bower) components. 
+
+# R
+## routing
+Routing is the process an app uses to determine how to respond to a client request for a particular endpoint. A default route is the endpoint the app directs the user to when no route is specified. In a PWA, the [app shell](#app-shell) performs routing.
+
+# S
+## service worker
+A script that a web browser runs in the background. On first accessing a particular site, the service worker fetches a defined set of resources and caches them. Then, the service worker acts as a client-side proxy. It intercepts network requests for resources and, where possible, supplies the resources from the browser cache. 
+
+When a service worker is installed, apps may be able to be used offline in a meaningful way.
+
+## shell
+In the [app-shell](#app-shell) design pattern, the shell is the main code that boots the app and loads necessary resources. The shell usually includes the common UI for the app, and the [router](#routing), which loads resources needed for the current route.
+
+## static dependency
+A [dependency](#dependency) that is required by a given file, and whose import is hard-coded using `<link rel=”import”>` or `<script src=”...”>`. This dependency will always be loaded when the given file is loaded.
```

Comment:
Consider rephrasing to note that these are just two examples of how a static dependency can be written.

e.g. `<img src="cat.gif">` is also a static dependency